examples/gatsby/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nacelle-gatsby-example",
-  "version": "2.8.1-alpha.0",
+  "version": "2.9.0-alpha.0",
```

Comment:
Please make sure to just bump the version to `2.9.0` without the alpha before merging & publishing.

Reply:
Sure thing - I'll just select "minor release" from the Lerna publishing options.